crypto/x509/x509_vfy.c

```diff
@@ -2924,6 +2924,26 @@ static int get_issuer(X509 **issuer, X509_STORE_CTX *ctx, X509 *cert)
     return ok;
 }
 
+static int augment_stack(STACK_OF(X509) *src, STACK_OF(X509) **dstPtr)
+{
+    if (src) {
+        STACK_OF(X509) *dst;
+        int i;
+
+        if (*dstPtr == NULL && (*dstPtr = sk_X509_new_null()) == NULL)
```

Comment:
suggestion : `sk_X509_new_reserve(NULL, sk_X509_num(src)` instead of `sk_X509_new_null()`

Reply:
Fixed. Please reconfirm approvals.